From cb5755b3af6d7a4fef3a8924a8008d3cb90f700b Mon Sep 17 00:00:00 2001
From: "Zhou,Jianfengx" <jianfengx.zhou@intel.com>
Date: Mon, 30 Oct 2017 16:23:54 +0530
Subject: [PATCH 14/16] REVERTME: Enable ALLOW_PERMISSIVE_LINUX for user build

Modify ALLOW_PERMISSIVE_LINUX=1 for building user verison
to get into homescreen

Change-Id: I6a64040a08b9ecb1210eea5a1e769b93cae464f2
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-52208
---
 init/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Android.mk b/init/Android.mk
index 161256e..0128a5b 100644
--- a/init/Android.mk
+++ b/init/Android.mk
@@ -13,7 +13,7 @@ init_options += \
 else
 init_options += \
     -DALLOW_LOCAL_PROP_OVERRIDE=0 \
-    -DALLOW_PERMISSIVE_SELINUX=0 \
+    -DALLOW_PERMISSIVE_SELINUX=1 \
     -DREBOOT_BOOTLOADER_ON_PANIC=0 \
     -DDUMP_ON_UMOUNT_FAILURE=0
 endif
-- 
1.9.1

